model-optimizer/extensions/front/YOLO.py

```diff
@@ -51,12 +69,150 @@ def transform_graph(self, graph: Graph, replacement_descriptions):
                             'Refer to documentation about converting YOLO models for more information.'.format(
                     ', '.join(replacement_descriptions['entry_points']), input_node_name))
             last_node = Node(graph, input_node_name).in_node(0)
+            anchors = replacement_descriptions['anchors']
+
             op_params = dict(name=last_node.id + '/YoloRegion', axis=1, end_axis=-1, do_softmax=0)
             op_params.update(replacement_descriptions)
             if 'masks' in op_params:
                 op_params['mask'] = op_params['masks'][i]
+                anchors = anchors[op_params['mask'][0] * 2 : op_params['mask'][-1] * 2 + 2]
                 del op_params['masks']
             region_layer_node = RegionYoloOp(graph, op_params).create_node([last_node])
+
+            if not enable_postprocess:
+                region_layer_node.dim_attrs.remove('axis')
+                Result(graph, {'name': region_layer_node.id + '/Result'}).create_node([region_layer_node])
```

Comment:
missed `return`?

Reply:
There wasn't `return` for original function and this `if` works wihout `return`

Reply:
what do you mean by original function? At this moment we execute the code below this `if` despite `enable_postprocess`